src/main.cpp

```diff
@@ -7,9 +7,33 @@
 #include <QQmlApplicationEngine>
 #include <QCommandLineParser>
 #include <QQmlContext>
+#include <QSettings>
+#include <QFile>
 #include "device_info.hpp"
 #include "rauc.hpp"
 
+void WriteDefaultSettings()
+{
+    QSettings settings;
+    QString group = settings.group();
+    QList<QString> enabled_pages = {
+                                    "Image Viewer",
+                                    "Multimedia",
+                                    "RAUC – Update Client",
+                                    "Multitouch",
+                                    "Device Information",
+                                    "Widget Factory",
+                                    "About PHYTEC"
+    };
+
+    settings.beginWriteArray("enabled_pages", enabled_pages.size());
+    for (int i = 0; i < enabled_pages.size(); ++i) {
+        settings.setArrayIndex(i);
+        settings.setValue("name", enabled_pages.at(i));
```

Comment:
Please use camel case for C++ and QML variables and dashed variables for command line and configuration options:
```suggestion
    settings.beginWriteArray("enabled-pages", enabledPages.size());
    for (int i = 0; i < enabledPages.size(); ++i) {
        settings.setArrayIndex(i);
        settings.setValue("name", enabledPages.at(i));
```